buildenv/jenkins/openjdk_tests

```diff
@@ -275,21 +277,35 @@ def runTest() {
             def SCM_GIT_REPO_VAL = sh(script: "echo ${SCM_GIT_REPO}", returnStdout: true).trim()
             SCM_GIT_REPO_VAL = SCM_GIT_REPO_VAL.replace("https://github.com/","git@github.com:")
 
-            sh "git clone -b ${SCM_GIT_BRANCH} ${SCM_GIT_REPO_VAL} aqa-tests"
+            retry(5) {
+                if (retry_count > 0) {
+                    println("Sleep ${sleep_time} secs, then retry ${retry_count}...")
```

Comment:
`retry` and `sleep` already print msg (see below). I think we can remove the extra `println`.
```
Retrying
[Pipeline] {
[Pipeline] echo
Sleep 180 secs, then retry 4...
[Pipeline] sleep
Sleeping for 3 min 0 sec
 > git rev-parse --resolve-
````

Reply:
updated